src/test/java/com/wikia/webdriver/testcases/activityfeedstests/WikiActivityTests.java

```diff
@@ -104,4 +111,72 @@ public void WikiActivityTests_004_newCategorizationIsRecordedOnActivityModule()
         .open()
         .verifyRecentNewCategorization(articleName, credentials.userName);
   }
+
+  @Test(groups = "WikiActivity_005")
+  @Execute(asUser = User.USER)
+  public void WikiActivityTests_005_newEditionWoVisualChangeNotRecordedOnActivityModule() {
+    String articleTitle = "newEditionWoVisualChangeNotRecordedOnActivityModule";
+    new ArticleContent().push("content", articleTitle);
+
+    ArticlePageObject article = new ArticlePageObject().open(articleTitle);
+    String articleName = article.getArticleName();
+    String articleContent = article.getContent();
+    VisualEditModePageObject visualEditMode = article.navigateToArticleEditPage();
+    visualEditMode.addContent(articleContent);
+    visualEditMode.submitArticle();
+    article.verifyContent(articleContent);
+
+    new SpecialWikiActivityPageObject(driver)
+        .open()
+        .verifyNoRecentEditionIsPresent(articleName, credentials.userName);
+  }
+
+  @Test(groups = "WikiActivity_006")
+  @Execute(asUser = User.USER)
+  public void WikiActivityTests_006_clickingTitleRedirectsToArticle() {
+
+    SpecialWikiActivityPageObject activityPage = new SpecialWikiActivityPageObject(driver);
+    activityPage.open();
+    List<Activity> activityList = activityPage.getActivities(10);
+
+    String title = activityList.stream().findFirst().get().getTitle();
+    ArticlePageObject article = activityList.stream().findFirst().get().clickOnTitle();
+    String articleName = article.getArticleName();
+    Assertion.assertEquals(articleName, title);
+  }
+
+  @Test(groups = "WikiActivity_008")
+  @Execute(asUser = User.USER)
+  public void WikiActivityTests_008_clickingUserRedirectsToUserPage() {
+    String articleTitle = "clickingUserRedirectsToUserPage";
+    new ArticleContent().push("content", articleTitle);
+
+    ArticlePageObject article = new ArticlePageObject().open(articleTitle);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "article" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)